pkg/tsdb/grafana-postgresql-datasource/sqleng/sql_engine.go

```diff
@@ -108,7 +108,6 @@ func (e *DataSourceHandler) TransformQueryError(logger log.Logger, err error) er
 	// for security purposes.
 	var opErr *net.OpError
 	if errors.As(err, &opErr) {
-		logger.Error("Query error", "err", err)
```

Comment:
Removed this error logging as it is duplicate. We are going to log the error anyway.